cscs-checks/system/slurm/slurm.py

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 import reframe as rfm
-import reframe.utility.os_ext as ext
+import reframe.utility.osext as ext
```

Comment:
Same for this test

Reply:
Actually this is used, but I will change the `as ext` to `as osext`.